kie-server-parent/kie-server-tests/kie-server-integ-tests-drools/src/test/resources/logback-test.xml

```diff
@@ -8,7 +8,7 @@
     </encoder>
   </appender>
 
-  <logger name="org.kie.server" level="INFO" />
+  <logger name="org.kie.server" level="DEBUG" />
```

Comment:
@baldimir Doesn't seem like a good idea. The tests now produce hundreds of megabytes of output [1]. This is not acceptable as the default behavior. I guess you've committed this by mistake?

[1] I can see 250 MB on the PR build that is [currently running](https://rhba-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/KIE/job/master/job/pullrequest/job/droolsjbpm-integration-pullrequests/442/console) and it hasn't finished yet. For reference, other PR builds for this repo produce only 34 MB of console output.

Reply:
I might be wrong. I seem to get tons of logging output locally even with this level reverted to info.